plugin-server/src/main/ingestion-queues/kafka-queue.ts

```diff
@@ -176,6 +178,23 @@ export class KafkaQueue {
         } catch {}
     }
 
+    private addMetricListeners() {
+        const listenEvents = [
+            this.consumer.events.GROUP_JOIN,
+            this.consumer.events.CONNECT,
+            this.consumer.events.DISCONNECT,
+            this.consumer.events.STOP,
+            this.consumer.events.CRASH,
+            this.consumer.events.REBALANCING,
+            this.consumer.events.RECEIVED_UNSUBSCRIBED_TOPICS,
+            this.consumer.events.REQUEST_TIMEOUT,
+        ]
+
+        listenEvents.forEach((event) => {
+            this.pluginsServer.statsd?.increment('kafka_queue_consumer_event', { event })
+        })
```

Comment:
I don't think this register the `this.consumer.on()` handlers? Looks like just a loop over the array of event names